iocore/net/SSLUtils.cc

```diff
@@ -2065,6 +2062,54 @@ get_sni_addr(SSL *ssl)
   return sni_addr;
 }
 
+std::string
+get_verify_str(SSL *ssl)
+{
+  std::string verify_str;
+
+  SSLNetVConnection *netvc = SSLNetVCAccess(ssl);
```

Comment:
SSLNetVCAccess 😞 I don't see it as a blocker because QUIC connection to origin is probably years away, but we should try to avoid using it for new stuff.

Reply:
Yes, we probably could store something in the `ex_data` of the `SSL` object.